lib/schmobile/middleware.rb

```diff
@@ -31,7 +31,7 @@ def redirect?(request)
         redirecting = false
       end
 
-      redirecting ? redirect_location(request) : nil
```

Comment:
Nope, I was thinking of the filters. They need to be able to return `nil`, as far as I recall. This `#redirect?` is just weird.